### PR TITLE
Login validation

### PR DESCRIPTION
**Pull request by beatsbydan**

### CODE REVIEW

**Review by AdaezeIkemefuna — CHANGES_REQUESTED** [review]
Please add a screenshot of your implementation

**Comment by beatsbydan** [comment]
bandicam 2022-07-28 12-38-48-745
bandicam 2022-07-28 12-39-06-929

File: client/css/form.css [hunk 1]
```diff
@@ -204,18 +213,13 @@ form .inputControl {
   display: flex;
```
Comment (AdaezeIkemefuna) [inline client/css/form.css:213]: Add 
justify-content: flex-start;
 gap: 1px;

so the error icon and message are align properly.

**Review by AdaezeIkemefuna — APPROVED** [review]

**Comment by beatsbydan** [comment]
bandicam 2022-07-28 13-03-12-012
bandicam 2022-07-28 13-03-22-912

**Comment by AdaezeIkemefuna** [comment]
> Great

**Pull request merged**